tests/modules/format-utils.js

```diff
@@ -188,3 +189,59 @@ test('splitString', suite => {
         });
     });
 });
+test('unescapeEntities', suite => {
+    suite.test('given non-string input - throws TypeError', t => {
+        const inputs = [
+            new Set(),
+            new Map(),
+            {},
+            true,
+            undefined,
+            0,
+            () => {},
+        ];
+        t.plan(inputs.length);
+        inputs.forEach(input => t.throws(
+            () => unescapeEntities(input),
+            TypeError,
+            `should throw for ${typeof input}`,
+        ));
+    });
+    suite.test('given empty string - returns empty string', t => {
+        const inputs = [
+            '',
+            String(),
+        ];
+        t.plan(inputs.length);
+        inputs.forEach(input => t.deepEqual(
+            unescapeEntities(input),
+            '',
+            `should return empty for empty ${input.constructor.name}`,
+        ));
+    });
+    suite.test('given 1 string - returns item as string', t => {
+        const inputs = [
+            'string',
+            'test',
+            'King\'s Arms'
+        ];
+        t.plan(inputs.length);
+        inputs.forEach(input => t.deepEqual(
+            unescapeEntities(input),
+            input,
+            'should return what it is given when no escaped entities',
+        ));
+    });
+    suite.test('given escaped entities should return unescaped string', t => {
+        t.plan(1);
+        const input = 'King&#39;s Arms';
+        const expected = 'King\'s Arms';
+        t.deepEqual(unescapeEntities(input), expected, `sent '${input}' got ${expected}`);
```

Comment:
For a failed assertion, the reporter will display the actual and expected values, so the values don't need to be repeated in the message